### PR TITLE
Added Axioms to users

### PR DESCRIPTION
**Pull request by abhishektiwari**
Added Axioms to users (users.js)

### CODE REVIEW

**Comment by abhishektiwari** [comment]
Signed Facebook CLA.

File: website-1.x/data/users.js [hunk 1]
```diff
@@ -83,6 +83,13 @@ module.exports = [
     infoLink: 'https://ax.dev/',
     fbOpenSource: true,
     pinned: true,
+  },
+   {
+    caption: 'Axioms',
+    image: 'https://static.axioms.io/branding/favicon.svg',
```
Comment (slorber) [inline website-1.x/data/users.js:89]: To avoid future broken images we ask to add the image directly in the repo.

**Review by slorber — CHANGES_REQUESTED** [review]
Thanks!

LGTM, just add the image/svg to our repo instead of a remote url

**Review by lex111 — CHANGES_REQUESTED** [review]
Hi @abhishektiwari you are using v2 so you need to add your project to the proper file  - https://github.com/facebook/docusaurus/blob/master/website/src/data/users.js (because now you have added it to showcase for v1).

You will also need to upload a preview.

**Comment by slorber** [comment]
Ah good catch :) though it was a v1 site due to no dark mode toggle presence

**Comment by abhishektiwari** [comment]
@slorber @lex111 thanks. I will raise an updated pull request.

File: website/src/data/users.js [hunk 2]
```diff
@@ -26,6 +26,15 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    caption: 'Axioms',
```
Comment (slorber) [inline website/src/data/users.js:30]:
```suggestion
    title: 'Axioms',
```

caption was for v1

File: website/src/data/users.js [hunk 3]
```diff
@@ -26,6 +26,15 @@ const users = [
     fbOpenSource: false,
     pinned: false,
   },
+  {
+    title: 'Axioms',
+    description: 'Axioms Developer Hub and Documentation Portal',
+    preview: require('./showcase/axioms.png'),
+    infoLink: 'https://developer.axioms.io/',
```
Comment (slorber) [inline website/src/data/users.js:33]:
```suggestion
    website: 'https://developer.axioms.io/',
```

**Pull request merged**